Support latest changes in OWL AXIOM Reference Set RF2 specificiation

mucst opened the pull request.

Commit:
SO-3141 update export path of axiom refsets in rf2 archive

Commit:
SO-3141 refsets can be found in Terminology folder too in rf2 files

cmark commented on snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/rf2/exporter/Rf2RefSetExporter.java:
```diff
@@ -93,7 +93,13 @@ public Rf2RefSetExporter(final Rf2ReleaseType releaseType,
 		if (folder != null) {
-			return Paths.get(releaseType.toString(), SnomedRF2Folder.REFSET.getDisplayName(), folder.getDisplayName());
+			
+			if (refSetType == SnomedRefSetType.OWL_AXIOM) {
```
`OWL_ONTOLOGY` should go to the terminology folder as well.

cmark commented on snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/rf2/exporter/Rf2RefSetExporter.java:
```diff
@@ -93,7 +93,13 @@ public Rf2RefSetExporter(final Rf2ReleaseType releaseType,
-			return Paths.get(releaseType.toString(), SnomedRF2Folder.REFSET.getDisplayName(), folder.getDisplayName());
+			
+			if (refSetType == SnomedRefSetType.OWL_AXIOM) {
+				return Paths.get(releaseType.toString(), SnomedRF2Folder.TERMINOLOGY.getDisplayName());
```
And I think the path should not contain the `releaseType`. @nagyo ?

Commit:
SO-3141 owl antology refsets are in Terminology folder too...

Rf2RefSetExporter should not collect such urls from rf2 files that are
in Terminology folder and do not contain der2_sRefset in their name

cmark reviewed (approved):
LGTM!
